source/CommonJobs/CommonJobs.Application.EvalForm/EmployeeSearching/EmployeeToEvaluate_Search.cs

```diff
@@ -0,0 +1,96 @@
+﻿using CommonJobs.Domain.Evaluations;
+using Raven.Abstractions.Indexing;
+using Raven.Client.Indexes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace CommonJobs.Application.EvalForm.EmployeeSearching
+{
+    public class EmployeeToEvaluate_Search : AbstractMultiMapIndexCreationTask<EmployeeToEvaluate_Search.Projection>
+    {
+
+        public class Projection
+        {
+            public string Id { get; set; }
+            public string UserName { get; set; }
+            public string FullName { get; set; }
+            public string CurrentPosition { get; set; }
+            public string Seniority { get; set; }
+            public string Period { get; set; }
+            public string[] Evaluators { get; set; }
+            public string Responsible { get; set; }
+            public bool AutoEvaluationDone { get; set; }
+            public bool ResponsibleEvaluationDone { get; set; }
+            public bool CompanyEvaluationDone { get; set; }
+            public bool OpenToDevolution { get; set; }
+            public bool Finished { get; set; }
+        }
+
+        public EmployeeToEvaluate_Search()
+        {
+            AddMap<EmployeeEvaluation>(evaluations =>
+                from evaluation in evaluations
+                select new
+                {
+                    Id = evaluation.Id,
+                    UserName = evaluation.UserName,
+                    FullName = evaluation.FullName,
+                    CurrentPosition = evaluation.CurrentPosition,
+                    Seniority = evaluation.Seniority,
+                    Period = evaluation.Period,
+                    Evaluators = new dynamic[0],
+                    Responsible = evaluation.Responsible,
+                    AutoEvaluationDone = false,
+                    ResponsibleEvaluationDone = false,
+                    CompanyEvaluationDone = false,
+                    OpenToDevolution = evaluation.OpenToDevolution,
+                    Finished = evaluation.Finished
+                });
```

Comment:
> However, if you convert this into an Enum and somehow update the enum (since that's part of the model), you may not be able to easily translate the enum into these properties... unless you keep track of all of them, in which case the Enum is just redundant, or a calculated value.

I think that result object is clearly for read only, it is a projection and cannot be stored in DB, the enum in a convenient value to simplify result mappings and searches.